### ISSUE TITLE
Feature request: key shortcut to toggle between light/dark

### ISSUE THREAD

**Issue by cryog0at** [darkreader/darkreader#1957]
Super useful extension, usually the first I download when I get a new computer. One thing that would make it even better is if there was a hotkey toggle between light and dark mode, instead of having to switch the extension off altogether.

**Comment by counterbeing** [comment]
I think what you need exists. On a mac option-shift-a toggles the extension on and off for a particular site. You can toggle it for all sites with option-shift-d.

**Comment by counterbeing** [comment]
It actually says so right in the extension popup 
![image](https://user-images.githubusercontent.com/239798/74090950-2e7d2480-4a6f-11ea-8948-f393d485e29d.png)

**Comment by cryog0at** [comment]
that's not exactly what I'm talking about, the toggle current site seems to turn off the extension for that site but keeps it active for other sites that are open. what I'm talking about is a toggle between light and dark from within the extension, I don't see a way to do that without using the mouse.
![image](https://user-images.githubusercontent.com/30218888/74091091-b79d5700-4a81-11ea-9b2a-850421a2cdf5.png)

**Comment by counterbeing** [comment]
Maybe I’m still not understanding, but it sounds like what you want is the option-shift-d command. This toggle light and dark mode within the extension, affecting all sites. 

I’m not a developer on this project, but if it’s not clear to me, it’s not clear to them either. If this doesn’t solve your issue, maybe you could create an explanation with a set of steps that tell them what your expected behavior would actually be.

**Comment by cryog0at** [comment]
the light mode is not exactly white, it's more sepia, and I still get the emboldened text from the text stroke option. it's much more readable than turning off the extension altogether.

**Comment by counterbeing** [comment]
Ah yes. I misunderstood then. I see what you mean!

I confused dark and light mode with just on and off. :)

**Comment by cryog0at** [comment]
Extension off:
![image](https://user-images.githubusercontent.com/30218888/74091214-2af39880-4a83-11ea-9023-e10023302a09.png)

Extension on light mode
![image](https://user-images.githubusercontent.com/30218888/74091230-4363b300-4a83-11ea-88f6-dbface3ff151.png)

**Comment by cryog0at** [comment]
haha all good

**Comment by jasonmj** [comment]
Agreed, it would be nice to have a keyboard shortcut for toggling between light and dark modes. +1

**Comment by mpolidori** [comment]
I'd love this feature as well. Specifically, so that I can toggle between modes and still have custom color scrollbars, text highlight color, etc.
